tests/test_big_int_id.py

```diff
@@ -0,0 +1,71 @@
+from copy import deepcopy
+from sqlalchemy import inspect
+from sqlalchemy.types import BIGINT
+
+from gdc_ng_models.snacks import database as db
+from gdc_ng_models.models import (
+    misc,
+    notifications,
+    qcreport,
+    redaction,
+    released_data,
+    reports,
+    studyrule,
+    submission,
+)
+
+
+TARGETED_TABLE_NAMES = [
+    'filereport',
+    'notifications',
+    'qc_validation_results',
+    'qc_test_runs',
+    'redaction_log',
+    "released_data_log",
+    'gdc_reports',
+    "study_rule",
+    'transaction_logs',
+    'transaction_documents',
+]
+
+
+def test_biginteger_id_column(db_configs):
```

Comment:
Testing the behavior that we expect is better than testing the specific implementation.  In this case, we want our table ids to accept a large number.  To test that behavior, we could insert a row into each of these tables and have the id be one trillion.  For the test to succeed, we would expect that the row is inserted.

PostgreSQL has multiple types that could satisfy this requirement.  For example, `bigint`, `bigserial`, and `numeric`.  Our code doesn't care how it's implemented at the database level. The code only cares about the behavior.